### PR TITLE
Update inventory_publishers.csv

### PR DESCRIPTION
**Pull request by tdlowden**
# Pull Request

Related to [LINK TO ISSUE]

## PR TASKS

- [ ] The actual code changes.
- [ ] Tests written and passed.
- [ ] Any changes to docs?
- [ ] Any new
[requirements versions](https://github.com/GSA/inventory-app/blob/main/requirements.in.txt)
to pull in?

### CODE REVIEW

**Comment by FuhuXia** [comment]
If there are commas in the name, you need to double quote the name.

File: config/data/inventory_publishers.csv [hunk 1]
```diff
@@ -165,6 +165,7 @@ nrcs-usda-gov,Department of Agriculture,Natural Resources Conservation Service,,
 nrel-gov,Department of Energy,National Renewable Energy Laboratory,,,,
 ntia-doc-gov,Institute for Telecommunication Sciences,,,,,
 nuclear-regulatory-commission,Nuclear Regulatory Commission,,,,,
+office-of-environmental-management-los-alamos-department-of-energy,Office of Environmental Management, Los Alamos, Department of Energy,Office of Environmental Management, Los Alamos, Department of Energy,,,,
```
Comment (FuhuXia) [inline config/data/inventory_publishers.csv:168]:
```suggestion
office-of-environmental-management-los-alamos-department-of-energy,"Office of Environmental Management, Los Alamos, Department of Energy",,,,,

```

**Comment by FuhuXia** [comment]
oops, i meant to add a comment, not a commit.

**Comment by tdlowden** [comment]
Ah right. thanks. will adjust in a bit

**Review by FuhuXia — APPROVED** [review]

**Pull request merged**